spec/javascripts/app/views/help_view_spec.js

```diff
@@ -134,4 +141,28 @@ describe("app.views.Help", function(){
       });
     });
   });
+
+  describe("chat section", function(){
+    describe("chat enabled", function(){
+      beforeEach(function(){
+        gon.chatEnabled = true;
```

Comment:
In the help view you set `chat_enabled` on initialization. Because of that it doesn't change when you change `gon.chatEnabled` afterwards.

Add `this.view = new app.views.Help();` after this line to fix that.